src/sprites.asm

```diff
@@ -625,7 +625,26 @@ GradeRendering::
     ld [wGrade0+2], a
     ld [wGrade1+2], a
 
-    ; Is the grade S1 or better?
+    ; If the effect timer is something higher than 0, decrease it, if it is 0, make it $1e and increase the palette value
+    ; And check if the palette is 7, too.
+    ld a, [wEffectTimer]
+    cp a, $0
+    jr z, :+
```

Comment:
Please use names for labels. Anonymous labels will lead to hard to find bugs when stuff is added to this code.